### PR TITLE
View

### PR DESCRIPTION
**Pull request by zakiyousufi**
## Hi there 👋🏻
**Thanks for taking the time to review the project. Here are some things important to note about it:**

_Now into the project..._
### Changes implemented in the "App View" branch
✅ Implemented the four required views, following the wireframes and extracting specific data for each view.
✅ Updated test cases to meet new project changes, following the TDD approach.
✅ Used partials to isolate reusable pieces of code.
✅ Used previously defined methods to meet project expectations.
____________________________________________________________________________________________________________________________________
Final notes 📔
Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

**Review by cynthiainga — COMMENTED** [review]
Hi @zakiyousufi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn’t have the `Future  Features` section. Please kindly make sure to add the section **Future Features** to your README file, including 1-3 potential features to be added. This section come just after the `Author` section 🙃 😉 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by zakiyousufi** [comment]
@cynthiainga 
Thanks for your review and marked the missing part in README.md file

File: config/database.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@
 #
 default: &default
   adapter: sqlite3
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
```
Comment (andersonlebon) [inline config/database.yml]: - [ ] Kindly make sure you are using a PostgreSQL database as it's required for this project.

File: README.md [hunk 2]
```diff
@@ -133,6 +136,10 @@ Give a ⭐️ if you like this project!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🙏 Acknowledgments
+
+> Give credit to everyone who inspired your codebase
+
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
```
Comment (andersonlebon) [inline README.md:146]: - [ ] Kindly provide a license link for your project.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @zakiyousufi 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (zakiyousufi) [reply README.md:146]: @andersonlebon  
Thanks for your review and feedback I will add the marked parts

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @zakiyousufi  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Made changes requested by the previous reviewer 👍 
- Linter checks are passing ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zakiyousufi** [comment]
Hi @Lucas-Masaba 
Thanks for your kindness Review

**Pull request merged**